client/js/lounge.js

```diff
@@ -17,7 +17,6 @@ $(function() {
 		"/notice",
 		"/op",
 		"/part",
-		"/query",
```

Comment:
Well not exactly an alias for /msg, as some clients don't open a query window for /msg, and just show the message sent/received in the server window and only show a query window when you do /query.

But certanarily, I don't like that, so it's essentially the same thing. The important thing is that it opens a query window, but does /query still do a whois by default, or do you have to do /whois for that?